MISRA.md

```diff
@@ -31,6 +31,44 @@ _Ref 8.4.1_
         a declaration in header file is not useful as the assembly code will
         still need to declare it separately.
 
+
+#### Rule 11.5
```

Comment:
Since we are suppressing this Rule by comments, we should remove this rule from the deviations list ?

Reply:
This rule will be removed from deviations list in coverity project PR.